### DISCUSSION TITLE
New, Non-RGI/custom (ZWJ-)sequences

### DISCUSSION THREAD

**Discussion by ghost** [C1710/blobmoji#85] [Ideas]
I need new ZWJ-sequences in this font.

**Comment by C1710** [comment]
Which ones do you mean? As far as I can see, all sequences up to 13.1 are present...

**Comment by ghost** [comment]
A 😀+🎧 so it can equal the blob emoji to wear headphones.

**Comment by C1710** [comment]
> 
> 
> A 😀+🎧 so it can equal the blob emoji to wear headphones.

Something like that could be done once extension packs are finished, but these things are rather low priority for now and actually work better as a discussion...

**Reply by ghost** [reply]
When are the extension packs gonna be made and released?

**Reply by C1710** [reply]
I don't know. Technically I already have implemented most the functionality, but I haven't tested it yet...

**Reply by ghost** [reply]
What are all of the emojis in this pack?

**Reply by C1710** [reply]
None. There are currently no extension packs yet or whatsoever.  
It's just a technical feature added to the build tool which allows to choose sets of emojis to include in the font, not more, not less.

It's only one possible usecase to introduce custom new sequences; another usecase would e.g. be to just bundle different versions together to tidy everything up a bit.  
Another usecase would be to use e.g. Noto Emoji as a basis when they have new emojis, which are not included here yet.

So, there are no concrete extension packs with any emojis planned at the moment, it's just that these custom sequences will not be included into the font until such that mechanism is implemented (if at all).

**Reply by ghost** [reply]
And what does RGI mean?

**Reply by C1710** [reply]
Recommended for General Interchange, i.e. the emojis/sequences that are officially approved by Unicode
